packages/datadog-plugin-aws-sdk/src/helpers.js

```diff
@@ -30,6 +30,18 @@ const helpers = {
     span.finish()
   },
 
+  isEnabled (serviceIdentifier, config, request) {
+    if (typeof config === 'boolean') {
+      return config
+    } else if (!config || typeof config !== 'object' || !services[serviceIdentifier]) {
```

Comment:
objects can't coerce to false, so you can skip the first one here.

Reply:
They can to undefined and null.